src/main/java/com/epam/university/java/core/task011/Task011Impl.java

```diff
@@ -1,44 +1,36 @@
 package com.epam.university.java.core.task011;
 
-import java.util.List;
 import java.util.LinkedList;
 import java.util.Arrays;
 import java.util.ArrayList;
 
 public class Task011Impl implements Task011 {
     @Override
     public String getLastName(String[] collection) {
-        List<String> names = new LinkedList<>();
+        ArrayList<String> names = new ArrayList<>(collection.length);
```

Comment:
if you are not going to use special methods of ArrayList, you can declare variable as List instead of ArrayList, ex:

```
List<String> list = new ArrayList<>();
```